### ISSUE TITLE
No Gyroscope / Motion controls

### ISSUE THREAD

**Issue by CookieJar0** [weihuoya/citra#854]
Playing on the 20231007 citra antutu build and there is no way to access motion controls, i saw a thing about previous builds which are outdated and perform poorly, and the gyroscope problem affects my mario and luigi dream team and kirby experiences badly

**Comment by Danixcrack2005** [comment]
I agree it also fails even when I use the camera or when I play Super María 3D Land I hope they fix it in the next update
